### DISCUSSION TITLE
Remember last selected option for "sign commit"

### DISCUSSION THREAD

**Discussion by ImoutoChan** [gitextensions/gitextensions#11504] [Q&A] [answered]
![image](https://github.com/gitextensions/gitextensions/assets/14975965/27e126b3-f8d8-4a86-b521-45c111b6e615)
 
Is there a way to tell git extensions to automaticly select this option for all commits in repo? Right now I need to select it manually for each commit

**Comment by vbjay** [comment, marked as answer]
Not in gui yet. But you can use git config to do so.

commit.gpgSign
tag.gpgSign
